[IV-23-24] Objetivo 6

LuciaAnsino opened the pull request:
- [x] ¿Se han establecido criterios a priori para elegir el sistema de
integración continua más conveniente y se han calificado los sistemas que
cumplan esos requisitos según esos criterios?

- [x] ¿Se han examinado varios sistemas de integración continua?

- [x] ¿Se han configurado varios sistemas de integración continua?

- [x] ¿Se ha configurado correctamente el Checks API en los sistemas en los que
sea necesario para que aparezca correctamente en GitHub (y se pueda comprobar desde los tests)?

- [x] ¿Uno de los sistemas configurados permite comprobar cuales son las versiones dellenguaje con las que funciona correctamente nuestra aplicación?

- [x] ¿Se escogen de forma adecuada las versiones del lenguaje que se testean,
 tanto en el sistema de CI como en el contenedor Docker?

- [x] ¿Se han justificado correctamente las versiones del lenguaje que se están
testeando y se ha comprobado que no se comprueban varias veces lo mismo (la misma versión en CI y en el contenedor Docker en otro sistema CI?

LuciaAnsino committed:
Elección de herramienta CI #31

COmento diferentes herramientas CI y elijo la que voy a usar closes #31

LuciaAnsino committed:
Añadir archivos de CI #31

Subo el archivo de la elección del CI closes #31

LuciaAnsino committed:
Github Actions #32

Configuro github actions closes #32

LuciaAnsino committed:
Arreglo error #32

Arreglo el fichero para que funcione github action closes #32

LuciaAnsino committed:
Arreglo error sintaxis #32

Arreglo sintaxis de instalación pnpm closes #32

LuciaAnsino committed:
Cambio la herramienta usada #31

Añado una herramienta CI más y cambio la eleccion de la herramientaa closes #31

LuciaAnsino committed:
COnfiguracion appveyor #34

Cnfiguración en appveyor.yml closes #34

LuciaAnsino committed:
Actualizo IV #34

Actualizo IV closes #34

LuciaAnsino committed:
Cambio de nombre

Cambio de nombre para quitar mayusculas

LuciaAnsino committed:
Cambio de nombre

Cambio de nombre para quitar mayusculas

MarioGuisado commented on appveyor.yml:
Deberías quitar los comentarios.

LuciaAnsino replied:
Resuelvo en [e6f2817](https://github.com/LuciaAnsino/CompraOnline/pull/33/commits/e6f2817bd300877afe9ff29a7f661cfa9567df2b)

LuciaAnsino committed:
Elimino comentarios

Elimino comentarios de appveyor.yml

LuciaAnsino commented:
Listo para revisión @JJ

JJ reviewed (changes requested):
Me temo que no estás testeando usando el contenedor en ninguno de los sistemas de CI configurados.
Creo, además, que estás testeando la versión 20 de node dos veces. Por favor, compruébalo.

LuciaAnsino committed:
Testeo en cada CI diferentes versiones

Con appveyor testeo la version estable y con GitHub Actions la version lts y al lanzar el docker se usa la dev

LuciaAnsino committed:
Uso docker en appveyor

Cambio appveyor para que uso el docker

LuciaAnsino committed:
Arreglo error de appveyor

Modificación de la configuración de appveyor para arreglar errores

LuciaAnsino committed:
Arreglo error de appveyor

Modificación de la configuración de appveyor para arreglar el error del directorio app/test

LuciaAnsino committed:
Arreglo error de appveyor

Arreglo sintaxis

LuciaAnsino committed:
Arreglo error de appveyor

Arreglo error de cierre de docker

LuciaAnsino committed:
Arreglo error de appveyor

Arreglo error de cierre de docker

LuciaAnsino committed:
Arreglo error de appveyor

Arreglo error de cierre de docker

LuciaAnsino committed:
Arreglo error de appveyor

Modificación de la configuración de appveyor para arreglar errores

LuciaAnsino committed:
Arreglo error de appveyor

Arreglo error de cierre de docker

LuciaAnsino committed:
Arreglo error de appveyor

Arreglo error de cierre de docker

LuciaAnsino committed:
Arreglo error de appveyor

Modificación de la configuración de appveyor para arreglar errores

LuciaAnsino committed:
Arreglo error de appveyor

Modificación de la configuración de appveyor para arreglar errores

LuciaAnsino committed:
Arreglo error de appveyor

Arreglo error de cierre de docker

LuciaAnsino committed:
Arreglo error de appveyor

Modificación de la configuración de appveyor para arreglar errores

LuciaAnsino committed:
Arreglo error de appveyor

Modificación de la configuración de appveyor para arreglar errores

LuciaAnsino committed:
Arreglo error de appveyor

Añado tiempo en el dockerfile para evitar que se cierre rápido

LuciaAnsino committed:
Arreglo error de appveyor

Modificación de la configuración de appveyor para arreglar errores

LuciaAnsino committed:
Arreglo error de appveyor

Añado tiempo en el dockerfile para evitar que se cierre rápido

LuciaAnsino committed:
Modifico el dockerfile a la versión anterior

MOdificio el dockerfile para dejarlo como estaba tras el objetivo 5

LuciaAnsino committed:
Modifico la configuración AppVeyor

Elimino linea erronea del fichero de configuración del appveyor

LuciaAnsino committed:
Arreglo configuración de Appveyor

Lanzo con appveyor la versión estable de node sin usar docker

LuciaAnsino committed:
MOdifico configuración de github action

Modifico la configuración para que use docker

LuciaAnsino commented:
@JJ ya he usado docker y solo testeo la verisón 20 una unica vez en AppVeyor, la versión dev es la que se usa por defecto en el docker y con Github Actions testeo la lts.

JJ reviewed (approved):
:+1:
